test/aggregate.js

```diff
@@ -1,9 +1,173 @@
 'use strict'
+const _ = require('lodash')
 const test = require('tape')
 const winnow = require('../src')
 const features = require('./fixtures/trees.json')
 const snowFeatures = require('./fixtures/snow.json')
 const budgetTable = require('./fixtures/budgetTable.json')
+const classBreaks = require('./fixtures/generateBreaks/generateRenderer-ClassBreaks.json')
+const uniqueValue = require('./fixtures/generateBreaks/generateRenderer-UniqueValue.json')
+
+test('create class breaks', t => {
+  t.plan(5)
+  const options = _.cloneDeep(classBreaks)
+  const results = winnow.query(features, options)
+  t.equal(Array.isArray(results), true)
+  t.equal(Array.isArray(results[0]), true)
+  t.equal(results.length, 5)
+  t.deepEqual(results[0], [0, 2.8])
+  t.deepEqual(results[4], [11.3, 14])
```

Comment:
I think its good to have at least one test that validates all of the individual breaks